xonsh/pyghooks.py

```diff
@@ -327,6 +330,28 @@ def __new__(cls, *args, **kwargs):
     return XonshStyleProxy
 
 
+def enhance_colors_for_cmd_exe():
+    # Test if we are on WINDOWS with CMD.exe 
+    # and PROMPT_TOOLKIT_STYLES is unset
+    env = builtins.__xonsh_env__
+    if 'CONEMUANSI' not in env and env.get('PROMPT_TOOLKIT_STYLES') is None:
+        # Only change the style if $PROMPT_TOOLKIT_STYLES is not touched and
+        # and we are not using ConEmu. 
+        smap = {Token.Name.Variable:'#44ffff',
+                Token.Generic.Prompt:'#44ffff',
+                Token.Name.Namespace:'#00aaaa',
+                Token.Name.Function:'#00aaaa',
+                Token.Name.Class:'#00aaaa',
+                Token.Generic.Heading:'#00aaaa',
+                Token.Literal.String.Symbol:'#00aaaa',
+                Token.Literal.String:'#ff4444',
+                Token.Name.Constant:'#ff4444',
+                Token.Keyword.Type:'#ff4444',
+                Token.Generic.Error:'#ff4444',
+                Token.AutoSuggestion:'#ansidarkgray'}         
```

Comment:
You can't do that. You have to use a conditional to check for `hasattr(pygments.style, 'ansicolors')` as the version of pygments that support `#ansidarkgray` has not been released yet.